build.cake

```diff
@@ -252,13 +252,23 @@ Task("Restore")
     .IsDependentOn("Setup")
     .Does(() =>
 {
-    var exitCode = StartProcess(dotnetcli,
+    Information("Restoring packages....");
+    var p = StartAndReturnProcess(dotnetcli,
         new ProcessSettings
         {
-            Arguments = "restore"
+            Arguments = "restore",
+            RedirectStandardOutput = true
         });
-    if (exitCode != 0)
+    p.WaitForExit();
+    var exitCode = p.GetExitCode();
+
+    if (exitCode == 0)
+    {
+        Information("Package restore successful!");
+    }
+    else
     {
+        Error(string.Join("\n", p.GetStandardOutput()));
```

Comment:
should we parse GetStandardOutput for any `warn:` items and emit those as information, when it passes.